packages/core/util/Base1DViewModel.ts

```diff
@@ -66,13 +66,23 @@ const Base1DView = types
     get minimumBlockWidth() {
       return 20
     },
-    bpToPx({ refName, coord }: { refName: string; coord: number }) {
+    bpToPx({
+      refName,
+      coord,
+      regionNumber,
+    }: {
+      refName: string
+      coord: number
+      regionNumber?: number
+    }) {
       let offsetBp = 0
 
-      const index = self.displayedRegions.findIndex(r => {
+      const index = self.displayedRegions.findIndex((r, idx) => {
         if (refName === r.refName && coord >= r.start && coord <= r.end) {
-          offsetBp += r.reversed ? r.end - coord : coord - r.start
-          return true
+          if (regionNumber ? regionNumber === idx : true) {
```

Comment:
can you add a comment here to the effect of why a regionNumber would be passed e.g. that it handles specifically the case of overlapping displayedRegions

Reply:
sure thing